Models/IPaymentRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Blahazon2._0.Models
```

Comment:
All repository code should be placed somewhere else e.g. a `Data` or `Persistence` namespace. Or even better to store the whole functionality in a separated class library.